CIP-0105/README.md

```diff
@@ -237,6 +237,134 @@ This is because we have modeled DRep IDs and CC credentials on stake pool operat
 
 The advantage of including a network tag would be to reduce the likelihood of mislabelling a DRep’s network of operation (eg Preview v Cardano mainnet).
 
+### Test vectors
+
+Scripts were constructed according to the following script templates:
+
+`script_1: all [keyhash, active_from 5001]`
+`script_2: any [keyhash, all [active_from 5001, active_until 6001]]`
+
+where
+
+`keyhash: {drep | cc_cold | cc_hot}`
+
+#### Test vector 1
+`mnemonic: test walk nut penalty hip pave soap entry language right filter choice`
+`accountIx: 0x80000000`
+```
+drep_xsk: drep_xsk14rjh4rs2dzm6k5xxe5f73cypzuv02pknfl9xwnsjws8a7ulp530xztarpdlyh05csw2cmnekths7dstq0se3wtza84m4fueffezsjfgassgs9xtfzgehrn0fn7c82uc0rkj06s0w0t80hflzz8cwyry3eg9066uj
+drep_sk:
+drep_xvk: drep_xvk17axh4sc9zwkpsft3tlgpjemfwc0u5mnld80r85zw7zdqcst6w543mpq3q2vkjy3nw8x7n8asw4es78dyl4q7u7kwlwn7yy0sugxfrjs6z25qe
+drep_vk: drep_vk17axh4sc9zwkpsft3tlgpjemfwc0u5mnld80r85zw7zdqcst6w54sdv4a4e
+drep: drep15k6929drl7xt0spvudgcxndryn4kmlzpk4meed0xhqe25nle07s
+drep_script_1: drep_script16pjhzfkm7rqntfezfkgu5p50t0mkntmdruwlp089zu8v29l95rg
```

Comment:
```suggestion
```

the `drep_script` prefix is meant to be used for script-based DReps, including here I think is a mistake

Reply:
this is the key:
```
drep_script16pjhzfkm7rqntfezfkgu5p50t0mkntmdruwlp089zu8v29l95rg
```
`drep_script_1` means we used the following script template:
```
script_1: all [keyhash, active_from 5001]
```
meaning as `drep` is acting as verification key hash we construct the following script:
```
all [drep15k6929drl7xt0spvudgcxndryn4kmlzpk4meed0xhqe25nle07s, active_from 5001]
```
serialize it and hash it, and then outputting with bech32 HRP "drep_script".

Reply:
ah okay, I understand now - thank you for the help!